libs/voicefocus/voicefocus.js

```diff
@@ -255,6 +256,9 @@ class VoiceFocus {
     }
     createNode(context, options) {
         var _a;
+        if (this.internal.isDestroyed) {
+            throw new Error("Unable to create node because VoiceFocus worker has been destroyed.");
```

Comment:
Use single quotes. Do you have a corresponding TypeScript file that outputs to this JavaScript?